### ISSUE TITLE
Does not respect MagShowSystemCursor

### ISSUE THREAD

**Issue by Blinue** [moonlight-stream/nvidia-gamestream-issues#19]
I'm using [MagShowSystemCursor](https://docs.microsoft.com/en-us/windows/win32/api/magnification/nf-magnification-magshowsystemcursor) to hide the system cursor in [Magpie](https://github.com/Blinue/Magpie), but I still see the cursor in the stream. Is it possible to support this API?

**Comment by cgutman** [comment]
So the cursor does not appear on your host's screen but it does appear on the client video stream in Moonlight while streaming?

**Comment by Blinue** [comment]
Yes. I use this API to hide the cursor so that I can draw it elsewhere, but I get double cursors in moonlight. 
![Screenshot_20211212-130054_Moonlight](https://user-images.githubusercontent.com/34770031/145701044-2f13e5ce-0ba6-4c0f-9667-8f421375fbb5.jpg)

BTW, OBS Studio doesn't have this issue
